languages/en.ini

```diff
@@ -617,6 +617,7 @@ In This Collection = "In This Collection"
 in_collection_label = "In collection:"
 include_synonyms = "Expand results using synonyms"
 Index Terms = "Index Terms"
+index_unknown_datetime = "Unknown"
```

Comment:
There's an existing "Unknown" translation. You could either use that, or else use the console tool to copy language strings to populate all the language files with it.

Reply:
I would like to keep it different as it is used in specific context, so it may have different translations in some languages. So I copied the current translation to other language files. I manually removed the string from pt and pt-br translation, because it was obviously bad for this context (index_unknown_datetime = online does not look right)

Reply:
Sounds good. I've sent a note to our Portuguese translation volunteers to see if we can get this corrected -- thanks for bringing this to my attention. I suspect that this is an error that has existed since the early days of the project.